packages/experience/src/pages/MfaBinding/TotpBinding/SecretSection/index.module.scss

```diff
@@ -0,0 +1,33 @@
+@use '@/scss/underscore' as _;
+
+.secretContent {
+  @include _.flex-column(center);
+
+  .qrCode {
+    border: 1px solid var(--color-line-divider);
+    margin: _.unit(4);
+    min-height: 148px;
+    min-width: 148px;
```

Comment:
Why just min-width and min-height? Also is this size mobile friendly?

Reply:
Sorry for the legency code when I'm developing this feature.
Accroding to the design file, the size of this QR code should be `136px * 136px`.
Updated.